src/main/resources/elasticsearch.template.properties

```diff
@@ -0,0 +1,36 @@
+# ============ elasticsearch configuration for picturesafe-search =============
+#
+# picturesafe-search requires a configuration file named 'elasticsearch.properties' in the classpath,
+# in which the following default parameters can be overwritten.
+
+
+# --------------------------- Client configuration ----------------------------
+
+## List of host addresses to connect to (blank separated, format <hostname>:<port> or <ip>:<port>)
+#elasticsearch.hosts=localhost:9200
+
+## automatically discover nodes from a running Elasticsearch cluster and set them to an existing RestClient instance
+#elasticsearch.sniffer.enabled=false
+
+# --------------------------- Index configuration -----------------------------
+
+#elasticsearch.index.alias=default
+
+## Index name prefix (property 'elasticsearch.index.alias' will be used when empty)
+#elasticsearch.index.name_prefix=
+
+## Index name date format ('yyyyMMdd-HHmmss-SSS' will be used when empty)
+#elasticsearch.index.name_date_format=yyyyMMdd-HHmmss-SSS
+
+#elasticsearch.index.number_of_shards=1
+
+#elasticsearch.index.number_of_replicas=0
+
+##  Maximum number of fields in the index (empty = use elasticsearch default)
+#elasticsearch.index.fields_limit=1000
+
+## Maximum result window size (maximum number of hits in a search result)
+#elasticsearch.index.max_result_window=10000
+
+# --------------------------- Query configuration -----------------------------
```

Comment:
"Time zone of the machine picturesafe-search is running on."

Reply:
ok